realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -200,11 +200,25 @@ InputStream getAssetFile() throws IOException {
         return schemaMediator.getModelClasses();
     }
 
+    /**
+     * Returns the absolute path to where the Realm file will be saved.
+     *
+     * @return the absolute path to the Realm file defined by this configuration.
+     */
     public String getPath() {
         return canonicalPath;
     }
 
     /**
+     * Checks if the Realm file defined by this configuration already exists.
+     *
+     * @return {@code true} if the Realm file exists, {@code false} otherwise.
+     */
+    public boolean realmExists() {
```

Comment:
This method is useful in some cases, but I don't think that we should make this public API.
Without proper lock, this method does not guarantee anything.